### PR TITLE
Document we don't create PodDisruptionBudgets

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Part of https://github.com/stackabletech/issues/issues/435

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [x] Documentation added or updated
- [x] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: docs/modules/opa/pages/usage-guide/operations/pod-disruptions.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+= Allowed Pod disruptions
+
+For OPA clusters we don't deploy any PodDisruptionBudgets (PDBs), as there is one instance per Kubernetes node running.
```
Comment (maltesander) [inline docs/modules/opa/pages/usage-guide/operations/pod-disruptions.adoc:3]:
```suggestion
For OPA clusters, the operator does not deploy any PodDisruptionBudgets (PDBs), as there is one instance per Kubernetes node running (Daemonset).
```

**Review by maltesander — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,14 +7,16 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Default stackableVersion to operator version ([#467]).
+- (Don't) support PodDisruptionBudgets ([#480]).
```
Comment (maltesander) [inline CHANGELOG.md:10]:
```suggestion
- Document that we do not create PodDisruptionBudgets ([#480]).
```

**Review by maltesander — COMMENTED** [review]
LGTM!

**Comment by sbernauer** [comment]
Lol, I forgot to push my changes

**Review by maltesander — APPROVED** [review]
LGTM :)

**Pull request merged**